resources/schemas/project.json

```diff
@@ -16,9 +16,6 @@
     "name": {
       "type": "string"
     },
-    "description": {
-      "type": "string"
-    },
```

Comment:
@igorsechyn I don't you should be removing this.

Reply:
I thought project description is becoming a document content too. Document content has a project if reference